test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -167,22 +213,30 @@ observations:
           units: "degree"
           range: [ 0, 360 ]
 
-        - name: "MetaData/heightOfStation"
-          source: variables/heightOfStation
+        - name: "MetaData/sensorViewAngle"
+          source: variables/sensorViewAngle
+          longName: "Sensor View Angle"
+          units: "degree"
+
+        - name: "MetaData/stationElevation"
+          source: variables/stationElevation
           longName: "Altitude of Satellite"
           units: "m"
 
         - name: "MetaData/sensorChannelNumber"
           source: variables/sensorChannelNumber
           longName: "Sensor Channel Number"
 
-        - name: "MetaData/fractionOfClearPixelsInFov"
-          source: variables/fractionOfClearPixelsInFov
+        - name: "MetaData/fractionOfClearPixelsInFOV"
+          source: variables/fractionOfClearPixelsInFOV
           longName: "Fraction of Clear Pixels in a Field of View"
           units: "1"
           range: [0, 1]
 
-        - name: "ObsValue/spectralRadiance"
+#       The unit from BUFR is W m-2 sr-1 m -- this is radiance per wavenumber
+#       - name: "ObsValue/spectralRadiance"
+        - name: "ObsValue/radiance"
           source: variables/spectralRadiance
           longName: "IASI Spectral Radiance"
-          units: "W m-2 sr-1 m"
+          units: "W m-2 sr-1"
+
```

Comment:
@BenjaminRuston 
I set output radiance in IODA as `radiance`.  In fact, it should be `spectralRadiance`
This is because the UKMO implemented radiance related routine (under variable transform) requires the variable name `radiance` instead of `spectralRadiance`.  

In addition, the spectral radiance could be `per wavenumber`, `per wavelength` or `per frequency`.  The unit for each is different.  

For examples:
spectral radiance = radiance per wavenumber (W m-2 sr-1 m)
spectral radiance = radiance per wavelength (W m-2 sr-1 m-1)

These are all spectral radiance, but the units are different depending on how you define "spectral".  The one directly from BUFR is radiance per wavenumber (W m-2 sr-1 m)

The ioda convention table only accept spectral radiance in wavelength (2 m-2 sr-1 m-1).  

```
  # All known variables are described here.
  - Variable: [ "radiance" ]
    Dimensions: [ [ "Location", "Channel" ] ]
    Attributes: { units: "W m-2 sr-1" }
  - Variable: [ "spectralRadiance" ]
    Dimensions: [ [ "Location", "Channel" ] ]
    Attributes: { units: "W m-2 sr-1 m-1" }
  - Variable: [ "scaledSpectralRadiance" ]
    Dimensions: [ [ "Location", "Channel" ] ]
    Attributes: { units: "W m-2 sr-1 m-1" }
```
We probably need to expand the ioda convention for spectralRadiance to accept "per wavenumber" and "per frequency" as well. 

For now, I just leave "spectralRadiance" as "Radiance" and set the unit accordingly to pass the ioda validator check for valid variable name, dimension and unit.

We can discuss this with UKMO later and check with them if they agree to change the radiance to spectral radiance in [here](https://github.com/JCSDA-internal/ufo/blob/develop/src/ufo/variabletransforms/Cal_SatRadianceFromScaledRadiance.cc).